classylss/cosmology.py

```diff
@@ -41,24 +96,31 @@ def __init__(self,
             T_cmb=2.7255,
             Omega_b=0.022032/0.67556**2,
             Omega_cdm=0.12038/0.67556**2,
-            Omega_Lambda=None,
             N_ur=None,
             m_ncdm=[0.06],
             P_k_max=10.,
             P_z_max=100.,
             gauge='synchronous',
-            **kwargs # additional arguments from CLASS
+            sigma8=0.82,
+            n_s=0.9667,
+            nonlinear=False,
+            verbose=False,
+            extra={}, # additional arguments to pass to CLASS
```

Comment:
Because CLASS (annoyingly) allows variable names that are not valid Python variables names and can't be passed via **kwargs. Things like "ln10^{10}A_s" or "non linear" (although I thought nonlinear was common enough to add a custom flag). So if we want to support more general, advanced uses, we need to pass as a dict too...I don't love the solution but couldn't really think of a better way